<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Windfreak_qt</class>
 <widget class="QMainWindow" name="Windfreak_qt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windfreak2 USB controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonUpdate_freq">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>70</y>
      <width>196</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Freq (MHz)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>120</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to windfreak</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controler Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSerialBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="control_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_power">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>310</y>
      <width>66</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_freq">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>245</y>
      <width>436</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>34</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>140</y>
      <width>141</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Power</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="freq_box">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>61</y>
      <width>136</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>36.000000000000000</double>
    </property>
    <property name="maximum">
     <double>4400.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.001000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="power_box">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>135</y>
      <width>56</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_wavelength">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>390</y>
      <width>506</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 nm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>526</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0 THz</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_channel">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>535</y>
      <width>56</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_channel_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>535</y>
      <width>446</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wavemeter channel number (secondary)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>450</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Optical Freq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>385</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wavelength</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_4">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>235</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RF freq</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_optical_freq_5">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>300</y>
      <width>226</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RF power</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>195</y>
      <width>166</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Channel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="channel_box">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>195</y>
      <width>56</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans [unknown]</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonUpdate_power">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>135</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonUpdate_channel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
